modulefiles/build.jet

```diff
@@ -1,26 +1,29 @@
 #%Module#####################################################
-## global_chgres component - jet
+## chgres build module for Jet
 #############################################################
 
-module purge
 module load intel/18.0.5.274
+module load impi/2018.4.274
 module load szip/2.1
-module load hdf5/1.8.9
-module load netcdf/4.2.1.1
+module load hdf5/1.10.4
+module load netcdf/4.6.1
 
-module load ip/v3.0.0
-module load sp/v2.0.2
-module load w3emc/v2.2.0
 module load w3nco/v2.0.6
+module load w3emc/v2.2.0
+module load sp/v2.0.2
+module load ip/v3.0.0
 module load bacio/v2.0.2
 module load sigio/v2.1.0
+module load sfcio/v1.0.0
 module load nemsio/v2.2.3
 module load nemsiogfs/v2.0.1
-module load sfcio/v1.0.0
 module load gfsio/v1.1.0
 module load landsfcutil/v2.1.0
- 
-export NETCDF_INCLUDE="-I${NETCDF}/include"
-export NETCDF_LDFLAGS_F="-L${NETCDF}/lib -lnetcdf -lnetcdff -L${HDF5}/lib -lhdf5 -lhdf5_fortran"
+module load g2/v3.1.0
 
-set FCMP ifort
+# Use DTCs version of esmf v8.  POC Dom H.
+module use -a /lfs3/projects/hfv3gfs/GMTB/modulefiles/intel-18.0.5.274
+module load esmf/8.0.0
+export NETCDF="/apps/netcdf/4.6.1/intel/18.0.5.274"
```

Comment:
move this line to where NETCDF is loaded.

Reply:
Done.